.gitignore

```diff
@@ -113,26 +38,9 @@ ENV/
 env.bak/
 venv.bak/
 
-# Spyder project settings
-.spyderproject
-.spyproject
-
-# Rope project settings
-.ropeproject
-
-# mkdocs documentation
-/site
-
 # mypy
 .mypy_cache/
-.dmypy.json
-dmypy.json
-
-# Pyre type checker
-.pyre/
-
-# pytype static type analyzer
-.pytype/
 
-# Cython debug symbols
-cython_debug/
+# Editor
+.vscode/
+.idea/
```

Comment:
minor: include newline

Reply:
@daavoo  should I merge, or do you intend to include this suggestion?